app/adapters/application.js

```diff
@@ -27,16 +26,6 @@ export default JSONAPIAdapter.extend(DataAdapterMixin, {
     return pluralize(underscored);
   },
 
-  urlForFindRecord(id, modelName, snapshot) {
-    let url = this._super(...arguments);
-    const query = get(snapshot, 'adapterOptions.query');
-    if (query) {
-      url += `?${$.param(query)}`;
```

Comment:
I don't know what this function is for, but instead of `$.params` you can use [this](https://dev.to/mornir/jqueryparam-in-vanilla-javascript-59mn).

Reply:
I never saw it triggered, do not know for which reason it was build but don't see a reason to keep it